src/ert/_c_wrappers/enkf/enkf_state.py

```diff
@@ -21,15 +20,13 @@
 def _internalize_GEN_DATA(
     ensemble_config: EnsembleConfig, run_arg: RunArg
 ) -> LoadResult:
-    keys = ensemble_config.getKeylistFromImplType(ErtImplType.GEN_DATA)
-
     run_path = Path(run_arg.runpath)
     errors = []
     all_data = {}
-    for key in keys:
-        config_node = ensemble_config[key]
+    for key in ensemble_config.get_keylist_gen_data():
+        config_node = ensemble_config.getNode(key)
         filename_fmt = config_node.input_file
```

Comment:
mypy really doesn't like this code. I think you want to write something like `config_node: EnkfConfigNode = ensemble_config[key] # type: ignore`

Reply:
Good idea.